composer.json

```diff
@@ -15,15 +15,15 @@
     ],
 
     "require": {
-        "php":                  ">=5.3.6",
+        "php":                  ">=5.5.9",
```

Comment:
Please rollback the change.

We can always change once agreement is reached in https://github.com/minkphp/Mink/issues/787.